parser/parser.y

```diff
@@ -278,6 +278,7 @@ import (
 	trim		"TRIM"
 	rtrim 		"RTRIM"
 	ucase 		"UCASE"
+	unixTimestamp	"UNIX_TIMESTAMP"
```

Comment:
it's not a keyword or non-keyword.
https://dev.mysql.com/doc/refman/5.7/en/keywords.html